src/pages/createBill/addAccountStep/components/BankNameDrawer/index.tsx

```
@@ -45,8 +43,11 @@ function BankNameDrawer({ open, onClose, setBankName }: BankNameDrawerProps) {
               </S.BankButton>
             ))}
           </S.GridContainer>
+          <S.FadeOverlay />
         </S.DrawerBody>
+        <S.ButtonWrapper>
         <S.SubmitButton onClick={handleSubmitButtonClick}>확인</S.SubmitButton>
```

ESLint

[error] Replace `<S.SubmitButton·onClick={handleSubmitButtonClick}>확인` with `··<S.SubmitButton·onClick={handleSubmitButtonClick}>⏎············확인⏎··········`

(prettier/prettier)